packages/big-design/src/components/Input/Input.tsx

```diff
@@ -6,7 +6,7 @@ import { Chip, ChipProps } from '../Chip';
 import { FormControlDescription, FormControlError, FormControlLabel } from '../Form';
 
 import { StyledIconWrapper, StyledInput, StyledInputContent, StyledInputWrapper } from './styled';
-interface Props {
+export interface Props {
```

Comment:
We were getting this error inside the `styled.tsx` file along with some other files:
```
Exported variable 'StyledCounterInput' has or is using name 'Props' from external module "/Users/chancellor.clark/big-design/packages/big-design/src/components/Input/Input" but cannot be named.
```

The tl'dr of this is because we are using `declarations: true` in `tsconfig.json` so the compiler needs to be told that `Props` is part of the module `Input.tsx` - it can't read it from `export type InputProps = Props & React.InputHTMLAttributes<HTMLInputElement>;`